sdk/keyvault/keyvault/package.json

```diff
@@ -25,7 +25,7 @@
     "rollup-plugin-node-resolve": "^3.4.0",
     "uglify-js": "^3.4.9"
   },
-  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/packages/@azure/keyvault",
+  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault",
```

Comment:
@KarishmaGhiya can you please file another work item to go through and update these home pages to point to the root of the repo instead of the project specific folders?

Reply:
@weshaggard Let me know if  there should be more information in the issue - https://github.com/Azure/azure-sdk-for-js/issues/1998